src/components/InputField/InputField.jsx

```diff
@@ -0,0 +1,51 @@
+import {
+  ItemWrapp,
+  StyledLabel,
+  StyledInput,
+  Error,
+  Success,
+  PasswordReapetLable,
+} from '@/components/InputField/InputField.styled';
+
+const InputField = ({ props, formik }) => {
+  const renderValidationMessage = elem => {
+    const label =
+      elem.general === 'password'
+        ? 'Your password must be 8+ characters, with at least 1 number and 1 special symbol'
+        : '';
+    if (formik.errors[props.general]) {
+      return <Error id={props.general}>{formik.errors[props.general]}</Error>;
+    }
+    if (!formik.errors[props.general] && formik.touched[props.general]) {
+      return <Success id={props.general}>Field is not empty</Success>;
+    }
+    return <PasswordReapetLable>{label}</PasswordReapetLable>;
+  };
+
+  return (
+    <ItemWrapp>
```

Comment:
```suggestion
    <InputField>
```

Reply:
it's a matter just of naming? i've renamed the element.

Reply:
It seems to me we agreed on the naming convention for styled components. If I messed up smth just ignore it.

Reply:
Done